src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -46,6 +57,8 @@
 
     private static Logger LOG = LoggerFactory.getLogger(StreetEdge.class);
 
+    private EdgeDataFromGenericFile extraData;
```

Comment:
I'm not sure we want sandbox features to add more fields to `StreetEdge`. Since this is the class with the most instances (often millions) even tiny changes have a large-ish impact in terms of memory consumption.

Would it make sense to isolate the feature even more by storing the edge data in a `Map<EdgeId, EdgeDataFromGenericFile>` for all edges and then looking it up when calculating the cost?

Reply:
It OK